README.md

```diff
@@ -1 +1,225 @@
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Added index.html file]**
+- **[Added style.css file]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming Soon
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+
+In order to run this project you need:
+1. Web Browser.
+2. Code Editor.
+3. Git-smc.
+### Setup
+
+Clone this repository to your desired folder:
+Run this command:
+```sh
+  cd my-folder
+  git clone https://github.com/rinom013/Hello-Microverse
+```
+--->
+
+### Install
+
+IInstall this project with:
+Run this command:
+```sh
+  cd my-project
+  npm install
+```
+--->
+
+### Usage
+
+TTo run the project, execute the following command:
+Open index.html using live server extension.
+### Run tests
+>Coming soon
+
+### Deployment
+
+You can deploy this project using:
+>Coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/rinom013)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/md-moniruzzaman-ba193229/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+-  Add Header.
+-  Add Footer.
+-  Add more styles.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank Sudipta Majumder & Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="LICENSE"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly add LICENSE to your project files, since its link does not work. Please watch this [tutorial](https://www.youtube.com/watch?v=8Dm7XXvtqXI) to know how to add one, remember to use the `MIT` license.